src/adaptors/alpaca-finance/index.js

```diff
@@ -0,0 +1,81 @@
+const axios = require("axios");
+const utils = require('../utils');
+
+function formatSymbol(sourceName) {
+    const space = sourceName.indexOf(' ');
+    return `${sourceName.substring(space + 1)} (${sourceName.substring(0, space)})`;
+};
+
+async function apy(chain) {
+    const response = (await axios.get(
+        `https://alpaca-static-api.alpacafinance.org/${chain}/v1/landing/summary.json`
+    )).data.data;
+
+    const filteredStakingPools = response.fairLaunchStakingPools
+        .filter(p => !p.key.includes('debt'))
+    const fairLaunchStakingPools = filteredStakingPools.map(p => ({
+        pool: `${p.stakingToken.address}-staking`,
+        chain: chain == 'bsc' ? 'Binance' : utils.formatChain(chain),
+        project: 'alpaca-finance',
+        symbol: utils.formatSymbol(p.symbol),
+        tvlUsd: Number(p.tvl),
+        apy: Number(p.apy)
+    }));
+
+    const strategyPools = response.strategyPools.map(p => ({
+        pool: `${p.key}-strategy-pool`,
+        chain: chain == 'bsc' ? 'Binance' : utils.formatChain(chain),
+        project: 'alpaca-finance',
+        symbol: utils.formatSymbol(p.iuToken.symbol),
+        tvlUsd: Number(p.tvl),
+        apy: Number(p.apy)
+    }));
+
+    const farmingPools = response.farmingPools.map(p => ({
+        pool: `${p.key}-farming-pool`,
+        chain: chain == 'bsc' ? 'Binance' : utils.formatChain(chain),
+        project: 'alpaca-finance',
+        symbol: formatSymbol(p.sourceName),
+        tvlUsd: Number(p.tvl),
+        apy: (Number(p.farmRewardApr) + Number(p.tradingFeeApr)) / p.leverage
```

Comment:
coming back to this line for a last time. in order to scale this value to actual apy we will need to do the following:
```
apr = (Number(p.farmRewardApr) + Number(p.tradingFeeApr)) / p.leverage
compoundFrequency = 365 // for daily compounding
apy = ((1 + apr / 100 / compoundFrequency)**compoundFrequency - 1) * 100
```
makes sense?

if we do that trasnformation we end up getting exact apy values with leverage 1x:

<img width="1135" alt="Screenshot 2022-05-12 at 15 26 50" src="https://user-images.githubusercontent.com/74533329/168098496-bc5d049d-1f4d-4551-8b73-8ed2777be54f.png">